installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc

```diff
@@ -13,7 +13,15 @@ include::modules/upi-prerequisites-for-scaling-a-upi-cluster.adoc[leveloffset=+2
 include::modules/upi-limitations-for-scaling-a-upi-cluster.adoc[leveloffset=+2]
 include::modules/configuring-a-provisioning-resource-to-scale-user-provisioned-clusters.adoc[leveloffset=+1]
 include::modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc[leveloffset=+1]
-include::modules/upi-managing-existing-hosts-in-a-upi-cluster.adoc[leveloffset=+1]
-include::modules/upi-removing-hosts-from-a-upi-cluster.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow#configuring-host-network-interfaces-in-the-install-config-yaml-file_ipi-install-installation-workflow[Configuring host network interfaces]
```

Comment:
This page is talking about install-config, which may be confusing for UPI. Maybe https://77264--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster#preparing-the-bare-metal-node_ipi-install-expanding is the better location to link to?

Reply:
Updated as suggested.